### PR TITLE
Promote ApacheCon 2018

### PR DESCRIPTION
**Pull request by nacx**

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://e3c4eb6dab71f0f3c347-dc00c568d9399ba5ba6a95cfe81ad0f2.r86.cf5.rackcdn.com/ to review your changes.

**Comment by andreaturli** [comment]
Thanks @nacx, I've started a PR to promote apachacon but you've been quicker. Actually I ended up trying to revamp the entire website, bumping jekyll version and so on ... let's see what I can do

**Comment by nacx** [comment]
Cool @andreaturli! I've already merged the PR as it is good and it should not block you on the Jekyll bump.

File: _includes/navbar.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
                         <li><a href="http://www.apache.org/licenses/">License</a></li>
                         <li><a href="http://www.apache.org/foundation/contributing.html">Donations</a></li>
                         <li><a href="http://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
-                        <li><a href="http://www.apache.org/security/">Report a Vulnerability</a></li>
+                        <li><a href="http://www.apache.org/security/">Security</a></li>
```
Comment (demobox) [inline _includes/navbar.html]: @nacx Unrelated change - is this an update to the recommended links?

File: _layouts/home.html [hunk 2]
```diff
@@ -6,8 +6,11 @@
     <body>
         {% include navbar.html %}
         <div id="home-content" class="container">
-            
+
+            {% comment %}
```
Comment (demobox) [inline _layouts/home.html]: Is this intended as a TODO to remove after the event?

**Comment by demobox** [comment]
Thanks for taking care of this, @nacx!

File: _includes/navbar.html (hunk 1 above)
Comment (nacx) [reply _includes/navbar.html]: Yes, it is unrelated but trivial so I just included it to match the navigation link policy: https://www.apache.org/foundation/marks/pmcs#navigation

File: _layouts/home.html (hunk 2 above)
Comment (nacx) [reply _layouts/home.html]: Yes. I've preferred to leave it like that in case we want to add it back we already know where in the page it looked ok.
